### PR TITLE
[13.0][IMP] website_helpdesk_mgmt: New addon

### PR DESCRIPTION
**Pull request by victoralmau**
New addon

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT39652

### CODE REVIEW

File: website_helpdesk_mgmt/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/helpdesk-13-0/helpdesk-13-0-website_helpdesk_mgmt
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/282/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to define a ticket as an action of a website form.
```
Comment (pedrobaeza) [inline website_helpdesk_mgmt/README.rst]:
```suggestion
This module allows to define a ticket as an action of a website form, which means to create a hepldesk ticket when a person fills the form.
```

**Review by pedrobaeza — APPROVED** [review]
I can't test it in runboat as it doesn't work right now, but the code review seems OK

**Review by chienandalu — APPROVED** [review]
Tested. Thanks!

**Pull request merged**